WalletWasabi/Blockchain/Keys/HdPubKey.cs

```diff
@@ -84,7 +85,10 @@ public double AnonymitySet
 
 	[JsonProperty(Order = 4)]
 	public KeyState KeyState { get; private set; }
-
+	
+	[JsonProperty(Order = 5)]
+	public int? LatestSpentHeight { get; set; }
```

Comment:
Keys are not spent. Use something less misleading instead.

Reply:
Guys, are you really think `RetirementHeight` is not misleading and everyone will get it immediately by looking at the code? 👎 

Keys cannot be spent but the coins on that address can - it is trivial. You are seeking perfection and we end up with the most incomprehensible naming.  

@lontivero please add constructive comments. Suggest a better name. 

If there is none change it back to LatestSpentHeigh and add a comment - if it is not trivial for someone. 

BTW it means that the latest spent coin on this address was at that height - it is not misleading at all.